app/views/devise/mailer/_creator_confirmation_instructions.html.erb

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+
+<head>
```

Comment:
As mentioned somewhere, maybe Slack, I will extract the shell of this email in another PR so it can be reused for other email templates. I'll wait until the next email template creation/change.